examples/bri-3/src/bri/identity/bpiSubjects/api/subjects.controller.spec.ts

```diff
@@ -0,0 +1,46 @@
+import { BadRequestException } from '@nestjs/common';
```

Comment:
small thing but just reminder that we should maybe discuss if we should unit test agents or test them through controllers like this only, or both?

Reply:
Open for recommendations. For me this combinations seems reasonable:
1) Unit test controllers with mocked storage (covers all the business logic)
2) E2E test endpoints (tests the full system without mocking)